IonDotnet.Tests/Integration/VectorBad.cs

```diff
@@ -26,6 +26,7 @@ public class VectorBad
             "clobWithNonAsciiCharacter.ion",
             "clobWithShortLiteralBlockCommentAtEnd.ion",
             "clobWithShortLiteralInlineCommentAtEnd.ion",
+            "clobWithLongLiteralInlineCommentAtEnd.ion",
```

Comment:
Note: when this is merged with #9, we need to be careful that this file isn't re-added to the exclusion list (since it was removed in #9 and moved to a different location in this PR).

Reply:
Okay, I will keep an eye on it.